src/main/java/io/gatling/demostore/WebConfig.java

```diff
@@ -11,7 +11,8 @@
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
```

Comment:
Note: cannot comment on binary file :'(

Zip file contains `__MACOSX` directory.
Why a zip file?

Reply:
> Zip file contains __MACOSX directory.

Oops! 😅 Should be fixed now.

>  Why a zip file?

Mostly just for the convenience of downloading the collection and environment files together.
(A minor side benefit is that the `.zip` file is automatically served as a file download by Spring, with the appropriate `Content-Disposition` - of course there's certainly a way to do that for a `.json` file, it's just not the default.)

Reply:
Our source code can manage to generate the zip file from flat json files?

We will be able to see them in github and changes will be clearer, won't we?

Reply:
It could be done at build time (though it's probably not worth the time it would take me to find out how to do that cleanly with Maven ^^) or on-the-fly in a custom controller (shouldn't be hard)... I'm not entirely convinced it matters very much for _these_ files, but hey, I can always do that. 🤷

Reply:
Mmh... Actually, doing it at build time may not be that hard, using the Maven assembly plugin but configuring it to run at the `generate-resources` phase instead of the `package` phase like in all the typical examples. I'm going to try that, if only out of curiosity. ^^

Reply:
Done.